src/code/z_horse.c

```diff
@@ -41,49 +58,58 @@ typedef struct HorseSpawn {
     /* 0x0A */ s16 type;
 } HorseSpawn;
 
-void func_8006D0EC(PlayState* play, Player* player) {
+void Horse_SetNormal(PlayState* play, Player* player) {
```

Comment:
maybe "SpawnGameplay" for the opposite of a "cutscene" spawn? Initially I thought this was talking about normal vectors based on the name